netlify/functions/transDim.js

```diff
@@ -0,0 +1,167 @@
+            (emailStyling.header_background_colour || "000000") +
+            "; padding: 10px 10px 10px 10px; color: #" +
+            (emailStyling.header_text_color || "ffffff") +
+            '; font-family: Courier, monotype; font-size: 20px; text-align: center;">\n' +
+            (emailStyling.alternate_header_image ||
+            emailStyling.netlify_header_image
+               ? '<div id="message-header-logo">\n<img src="cid:logo" style="height: 50px; display: block;" height="50" />\n</div>\n'
+               : "") +
+            '<div id="message-header-text">\n' +
+            (emailStyling.header_text || "Web Enquiry") +
+            '\n</div>\n</div>\n<div id="message-main" style="padding: 25px 10px 25px 10px; width: 100%; margin: auto; max-width: 600px; display: block; color: #' +
+            (emailStyling.body_text_color || "000000") +
+            ';">\n' +
+            fields
+               .map(
+                  (field) =>
+                     `<p style="font-family: Verdana, sans-serif; font-size: 15px;">\n<strong>\n${
+                        field.label
+                     }: </strong>${
+                        field.value == ""
+                           ? "<em>Not answered</em>"
+                           : field.value
+                     }\n</p>\n`
+               )
+               .join("") +
+            '<p style="font-family: Verdana, sans-serif; font-size: 15px;">\n<strong>Message:</strong>\n</p>\n<p style="font-family: Verdana, sans-serif; font-size: 15px;">\n' +
+            enquirerMessage +
+            '\n</p>\n</div>\n<div id="message-footer" class="banner" style="display: block; background-color: #' +
+            (emailStyling.footer_background_colour || "000000") +
+            "; padding: 10px 10px 10px 10px; color: #" +
+            (emailStyling.footer_text_color || "ffffff") +
+            '; font-family: Courier, monotype, sans-serif; font-size: 12px;">\n<p style="text-align: center">Reply to this email to reply to the sender.</p>\n</div>\n</div>\n</div>\n</body>\n</html>' // HTML (styled and formatted) email
+      };
+
+      try {
+         console.log("try");
+         const res = await sendEmail(email);
+         console.log(res);
+         return {
+            statusCode: 200,
+            headers: {
+               /* Required for CORS support to work */
+               "Access-Control-Allow-Origin": transDimConfig.allow_domain,
+               // /* Required for cookies, authorization headers with HTTPS */
+               "Access-Control-Allow-Credentials": true,
+               "Access-Control-Allow-Headers":
+                  "Origin, X-Requested-With, Content-Type, Accept",
+
+               "Content-Type": "text/plain"
+            }
+         };
+      } catch (err) {
+         console.log("catch");
+         console.log(err);
+         return {
+            statusCode: err.responseCode,
+            headers: {
```

Comment:
As these header are reused 3 times is it worth pulling into a const?

Reply:
see earlier comment